t/TEST

```diff
@@ -119,35 +119,53 @@ $| = 1;
 #BEGIN { require '../lib/warnings.pm'; "warnings"->import() };
 
 # remove empty elements due to insertion of empty symbols via "''p1'" syntax
```

Comment:
This comment should be moved to line 128.

Reply:
Thanks, done.